src/codeview.jl

```diff
@@ -129,7 +137,7 @@ function cthulhu_typed(io::IO, debuginfo, src, rt, mi, iswarn, stable_code, inli
     end
     postprinter = iswarn ? InteractiveUtils.warntype_type_printer : IRShow.default_expr_type_printer
 
-    should_print_stmt = (iswarn || src isa IRCode || stable_code) ? Returns(true) : is_type_unstable
+    should_print_stmt = (iswarn || src isa IRCode || verbose) ? Returns(true) : is_type_unstable
```

Comment:
This is not anything this PR needs to address, but I just added a comment about `verbose` in #189. Do you agree that the sense seems flipped?